t/core/trusted-addresses.t

```diff
@@ -0,0 +1,219 @@
+#
```

Comment:
Need to test the situation where trusted-addresses contain multiple IP addresses

Reply:
and need test cases in plugins.